### DISCUSSION TITLE
Movies download but then disappear

### DISCUSSION THREAD

**Discussion by BeardedTek** [lardbit/nefarious#213] [Q&A] [answered]
I've been having this issue lately.  A movie will download, then moments after it finishes downloading, it almost immediately is deleted and removed from transmission.

It appears it is blacklisted, which I believe I may have done.  How can I unblacklist a torrent?

**Comment by BeardedTek** [comment, marked as answer]
I found a workaround:

1. Stop nefarious (docker-compose down)
2. Open db.sqlite3 file in docker volume (usually /var/lib/docker/volumes/nefarious_nefarious_db/db.sqlite3)
3. remove entry for movie/show from nefarious_torrentblacklist
4. Restart nefarious (docker-compose up -d)

Once I have time I'll see if I can make a PR to add viewing blacklists and remove them possible.  I'll have to learn the codebase a bit first, but I still want to move forward on the adding actors to the titles display as well one day.....

**Comment by lardbit** [comment]
Hey @BeardedTek,  I believe you stumbled on a bug regarding the fake video detection feature (https://github.com/lardbit/nefarious/pull/180 & https://github.com/lardbit/nefarious/pull/204) I added a couple months ago.  Someone else (https://github.com/lardbit/nefarious/issues/203) reported a similar issue and I think this is what you're seeing too.  I haven't been able to reproduce the issue yet, but I added the ability to toggle the feature on/off on the main settings page.

However, I just committed a change (https://github.com/lardbit/nefarious/pull/214) to disable it for everyone automatically until I can resolve whatever is going on.

You may find helpful the undocumented "admin" backend which you can manually edit records, e.g **torrentblacklist** entries.  
http://127.0.0.1:8000/admin/nefarious/torrentblacklist/

**Reply by BeardedTek** [reply]
Oh cool!!!!!  Love this project!

**Comment by lardbit** [comment]
Can you confirm if https://github.com/lardbit/nefarious/pull/214 automatically disabled your setting?

**Reply by BeardedTek** [reply]
It would appear so.  Things are running smoothly again!
